AmazonChimeSDKDemo/AmazonChimeSDKDemo/CaptionCell.swift

```diff
@@ -23,8 +25,34 @@ class CaptionCell: UITableViewCell {
         self.contentView.backgroundColor = caption.speakerName.isEmpty ? .lightGray : .none
         speakerNameLabel.text = caption.speakerName
         speakerNameLabel.accessibilityIdentifier = caption.speakerName
-        captionContentLabel.text = caption.content
         captionContentLabel.accessibilityIdentifier = "caption-\(indexPath.row)"
+        // Highlight identified/redacted PIIs in green.
+        let  coloredString = NSMutableAttributedString.init(string: caption.content)
+        caption.entities?.forEach { word in
+            var firstOccurance = caption.content.startIndex
+            while firstOccurance < caption.content.endIndex,
+                  let range = (caption.content).range(of: word, range: firstOccurance..<caption.content.endIndex),
+                  !range.isEmpty {
+                let index = caption.content.distance(from: caption.content.startIndex, to: range.lowerBound)
+                coloredString.addAttribute(
+                    NSAttributedString.Key.foregroundColor,
+                    value: UIColor.green,
+                    range: NSRange(location: index, length: word.count))
+                firstOccurance = range.upperBound
+            }
+        }
+        // Underline low confidence words in red. 
+        caption.items?.forEach { item in
+            let word = item.content
+            let hasLowConfidence = (item.confidence ?? 1.0) < threshold && item.confidence != 0
+            let isCorrectContentType = word.first != "[" && item.type != TranscriptItemType.punctuation
```

Comment:
can you make `"["` a well-named constant?

Reply:
will be updated.